pvacseq/lib/pipeline.py

```diff
@@ -289,10 +286,9 @@ def execute(self):
             os.unlink(symlink)
 
 
-        print("\n")
-        print("Done: pvacseq has completed. File %s contains list of filtered putative neoantigens" % self.final_path())
-        print("We recommend appending coverage information and running `pvacseq coverage_filter` to filter based on sequencing coverage information")
-        sys.stdout.flush()
+        status_message("\n")
+        status_message("Done: pvacseq has completed. File %s contains list of filtered putative neoantigens" % self.final_path())
+        status_message("We recommend appending coverage information and running `pvacseq coverage_filter` to filter based on sequencing coverage information")
```

Comment:
You're right. I just pushed a commit to make this only call `status_message` once.